tests/cli/workflow/mip/conftest.py

```diff
@@ -15,6 +27,209 @@ def mock_root_folder():
+        sample_info_data = {
+            "date": dt.datetime.now(),
+            "analysisrunstatus": "finished",
+            "analysis_date": dt.datetime.now(),
+            "case": case,
+            "mip_version": "latest",
+        }
+
+        sample_info_path = mip_case_dirs[case] / f"{case}_qc_sample_info.yaml"
+        qc_sample_info[case] = sample_info_path
+        with open(sample_info_path, "w") as fh:
+            yaml.dump(sample_info_data, fh)
+
+    return qc_sample_info
+
+
+@pytest.fixture(name="mip_configs")
+def make_mip_configs(
+    mip_case_ids: dict, mip_case_dirs: dict, mip_deliverables: dict, mip_qc_sample_info: dict
+) -> dict:
+    """Create config files for mip case"""
+
+    yaml = YAML()
+
+    # Storing paths in a list to be returned and used in later stages.
+
+    config_dict = {}
+
+    for case in mip_case_ids:
+        config = {
+            "email": "fake_email@notscilifelab.com",
+            "case_id": case,
+            "samples": "dummy_samples",
+            "is_dryrun": False,
+            "outdata_dir": str(mip_case_dirs[case]),
+            "priority": "medium",
+            "sampleinfo_path": str(mip_qc_sample_info[case]),
+            "sample_info_file": str(mip_qc_sample_info[case]),
+            "analysis_type": {mip_case_ids[case]["internal_id"]: "wgs"},
+            "slurm_quality_of_service": "medium",
+            "store_file": str(mip_deliverables[case]),
+        }
+        config_path = mip_case_dirs[case] / f"{case}_config.yaml"
+        config_dict[case] = config_path
+        with open(config_path, "w") as fh:
+            yaml.dump(config, fh)
+
+    return config_dict
+
+
+@pytest.fixture(name="_store")
+def populated_store(base_store: Store, mip_case_ids: dict, helpers) -> Store:
+    """Create and populate temporary db with test cases for mip"""
+
+    _store = base_store
+
+    # Add apptag to db
+
+    helpers.ensure_application_version(store=_store, application_tag="WGSA", application_type="wgs")
+
+    # Add sample, cases and relationships to db
+
+    for case in mip_case_ids:
+        family = helpers.add_family(
+            store=_store,
+            internal_id=case,
+            family_id=mip_case_ids[case]["name"],
+        )
+        sample = helpers.add_sample(
+            store=_store,
+            sample=mip_case_ids[case]["internal_id"],
+            data_analysis="mip",
+            customer_name="cust000",
+            application_tag="WGSA",
+            application_type="wgs",
+            gender="unknown",
+        )
+        helpers.add_relationship(store=_store, sample=sample, family=family, status="affected")
+
+    return _store
+
+
+@pytest.fixture(name="empty_housekeeper_api")
+def populate_mip_housekeeper(real_housekeeper_api: HousekeeperAPI) -> HousekeeperAPI:
+    """Empty housekeeper for testing store in mip workflow"""
+
+    _housekeeper_api = real_housekeeper_api
+
+    return _housekeeper_api
+
+
+@pytest.fixture(name="_tb_api")
```

Comment:
```suggestion
@pytest.fixture(name="tb_api")
```
Could you give this a name that tells more about what it is? Perhaps `populated_tb_api` or something

Reply:
Will do!